config/routes.rb

```diff
@@ -14,7 +14,7 @@
   resources :categories, only: [:index]
   resources :items, only: [:index, :show]
   resource :cart, only: [:show, :destroy, :create]
-  patch '/items/:id' => 'carts#update'
+  patch '/cart' => 'carts#update'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.